src/components/Tag/Tag.stories.tsx

```diff
@@ -0,0 +1,122 @@
+/* eslint-disable max-lines */
+import { select, text } from '@storybook/addon-knobs';
+import React from 'react';
+
+import { StoryContainer } from '../../common/HelperComponents/StoryContainer';
+import { Tag } from './Tag';
+import { TagColors, TagPositions, TagSizes } from './Tag.types';
+
+const description = () => `
+- - -
+
+  ### NOTE:
+  This component is available in the following variants:
+
+  - ✅ Standard
+
+With the following attribute status:
+
+  - **Attributes:**
+    - ✅ **Text:**
+    - **Size:**
+      - ✅ \`Standard\`
+      - ✅ \`Small\`
+    - **Color:**
+      - ✅ \`Primary\`
+      - ✅ \`Secondary\`
+      - ✅ \`Alert\`
+      - ✅ \`Success\`
+      - ✅ \`Link\`
+      - ✅ \`Warining\`
+    - **Border Position:**
+      - ✅ \`Default\`
+      - ✅ \`Left\`
+      - ✅ \`Right\`
+
+- - -
+`;
+
+const tagColors = [
+  'alert',
+  'link',
+  'primary',
+  'secondary',
+  'success',
+  'warning',
+];
+
+export default {
+  component: Tag,
+  parameters: {
+    componentSubtitle: 'A tag labels UI objects for quick recognition and navigation.',
+    docs: {
+      extractComponentDescription: description,
+    },
+  },
+  title: 'Components|Tag',
+};
+
+export const Default = () => (
+  <StoryContainer title='Standard'>
+    <Tag text='Lorem Ipsum' />
```

Comment:
Nos primórdios, tínhamos combinado de não usar mais Lorem Ipsum nos exemplos das sample. Aqui tem alguns textos bem massa sobre isso: 

- https://www.secretstache.com/blog/stop-using-lorem-ipsum/
- https://blog.prototypr.io/why-testing-with-real-content-is-better-than-lorem-ipsum-c7c79586ee72
-

Reply:
Uhmm, não sabia dessa. Valeu!